frontend/src/libs/components/sidebar/styles.module.scss

```
@@ -45,16 +45,9 @@
   cursor: pointer;
 }
 
-.icon-selected {
-  position: relative;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  background-color: var(--white);
-  border: 0;
-  border-radius: 50%;
+.icon-selected,
+.icon-container:hover {
```

We don't need the same effect on focus?

---

We used identical hover and select effects on items of meditation and chat sidebars, I used the same approach. Personally I don't know how the hover effect should look like)

---

```suggestion
.icon-container:hover, .icon-container:focus {
```

I meant just this, cause we supposed to have some effects when clicking with Tab for example

---

Oh okay, I added it